consoles/sshVirtsh.pm

```diff
@@ -542,17 +560,32 @@ __END"
 
         # set default boot delay
         $self->run_cmd(qq{echo 'bios.bootDelay = "10000"' >> $vmx}, domain => 'sshVMwareServer');
-
-        # inject cloud init metadata and userdata required for the image if there are any
-        my $ci_meta = $bmwqemu::vars{CLOUD_INIT_META};
-        my $ci_user = $bmwqemu::vars{CLOUD_INIT_USER};
-        my $ci_encoding = $bmwqemu::vars{CLOUD_INIT_ENCODING};
-
-        if ($ci_meta && $ci_user && $ci_encoding) {
-            $self->run_cmd(qq{echo 'guestinfo.metadata = "$ci_meta"' >> $vmx}, domain => 'sshVMwareServer');
-            $self->run_cmd(qq{echo 'guestinfo.metadata.encoding = "$ci_encoding"' >> $vmx}, domain => 'sshVMwareServer');
-            $self->run_cmd(qq{echo 'guestinfo.userdata = "$ci_user"' >> $vmx}, domain => 'sshVMwareServer');
-            $self->run_cmd(qq{echo 'guestinfo.userdata.encoding = "$ci_encoding"' >> $vmx}, domain => 'sshVMwareServer');
+        my $fb_tool = $bmwqemu::vars{FIRST_BOOT_CONFIG};
+
+        if ($fb_tool && $fb_tool ne 'wizard') {
+            my $encoding = 'gzip+base64';
+            if ($fb_tool =~ /combustion|ignition/) {
+                if ($bmwqemu::vars{FIRST_BOOT_COMBUSTION}) {
```

Comment:
You might have to call this `ASSET_*` so that it gets copied

Reply:
Isn't a clone of http://github.com/os-autoinst/os-autoinst-distri-opensuse/ always present on the worker when a job is being spawned? The idea is basically the same as here https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/tests/console/journal_check.pm#L20

Reply:
Ok, so you want the config inside the test distri and not as an asset?

Reply:
Yeah, it is already present in `data` and the test itself doesn't need to have the file, just the encoded content